CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+Contributing to homs
```

Comment:
Don't we use uppercase for this project name?

Reply:
I used repo name.